Lombiq.Hosting.Tenants.EmailQuotaManagement/Services/IQuotaService.cs

```diff
@@ -34,7 +34,17 @@ public interface IQuotaService
     void ResetQuota(EmailQuota emailQuota);
 
     /// <summary>
-    /// Saves the given quota.
+    /// Saves the given quota on reminder sent.
     /// </summary>
-    void SaveQuota(EmailQuota emailQuota);
+    void SaveQuotaReminder(EmailQuota emailQuota);
+
+    /// <summary>
+    /// Returns <see langword="true"/> if the reminder email should be sent.
+    /// </summary>
+    bool ShouldSendReminderEmail(EmailQuota emailQuota, int? currentPercentage = null);
+
+    /// <summary>
+    /// Returns the current quota usage percentage.
+    /// </summary>
+    int CurrentUsagePercentage(EmailQuota emailQuota);
```

Comment:
Prefix it with "Get" , and add it in an extension method instead, because a percentage calculation doesn't need to be part of the interface. However, add a method to get the `EmailQuotaPerMonth` value, and use that instead in `EmailSettingsQuotaFilter` so it doesn't go directly to the option behind the service.